examples/node/small_index.js

```diff
@@ -10,27 +10,44 @@ const client = new MeiliSearch(config)
 const uid = 'movies_test'
 
 const dataset = [
-  { id: 123, title: 'Pride and Prejudice', comment: 'A great book' },
+  { biduleidbidule: "ahaha", id: 123, title: 'Pride and Prejudice', comment: 'A great book' },
```

Comment:
bidule? 😇

Reply:
this file should not have been pushed I will change it back

Reply:
Removed as it is removed in the other PR as well and replaced with a more relevant one